doc/radosgw/notifications.rst

```diff
@@ -161,55 +162,81 @@ name of an existing topic and different endpoint values).
 
 Request parameters:
 
-- push-endpoint: URI of an endpoint to send push notification to
-- OpaqueData: opaque data is set in the topic configuration and added to all notifications triggered by the topic
-- persistent: indication whether notifications to this endpoint are persistent (=asynchronous) or not ("false" by default)
+- push-endpoint: the URI of an endpoint to send push notifications to.
+- OpaqueData: opaque data is set in the topic configuration and added to all
+  notifications that are triggered by the topic.
+- persistent: indicates whether notifications to this endpoint are persistent
```

Comment:
@zdover23 This and other _indicates whether_ entries (as well as line 165's push-endpoint entry) should perhaps be rewritten into complete sentences (if for no other reason than to match the OpaqueData entry and others).